drake/systems/plants/RigidBody.h

```diff
@@ -19,7 +24,37 @@ class DRAKERBM_EXPORT RigidBody {
  public:
   RigidBody();
 
+  /**
+   * An accessor for the name of the link that this rigid body reprsents.
+   *
+   * @return The name of the link that's modeled by this rigid body.
+   */
+  const std::string& GetName() const;
```

Comment:
These functions names start with capital letters, but most of the other functions in this file start with lower case letters. Is this the correct style for Drake?